bpfassets/perf_event/perf_event.c

```diff
@@ -71,6 +71,11 @@ BPF_ARRAY(cache_miss, u64, NUM_CPUS);
 // cpu freq counters
 BPF_ARRAY(cpu_freq_array, u32, NUM_CPUS);
 
+#ifndef SAMPLE_RATE
+#define SAMPLE_RATE 1000
```

Comment:
Isn't skipping 1000 sample to extreme?
Did you try 10 and 100?

Reply:
yes, 10 or 100 does have some reduction from 1000ns to 300ns.